webapp/app/chart.php

```diff
@@ -0,0 +1,124 @@
+<?php
+  $data = $hours_each_day;
+  $Y_m = date("Y-m", strtotime("0 month"));
```

Comment:
```suggestion
  $y_m = date("Y-m", strtotime("0 month"));
```
大文字から始まると定数？見えるがあるので小文字スタートが良いかと

Reply:
修正済み